### PR TITLE
tests added first commit

### PR DESCRIPTION
**Pull request by DmytroSvirsa**

### CODE REVIEW

File: tests/test_taxi.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+        self.assertContains(response, self.driver.license_number)
+
+    def test_additional_info_fields_in_driver_add(self):
+        url = reverse("admin:taxi_driver_add")
+        response = self.client.get(url)
+
+        self.assertContains(response, "first_name")
+        self.assertContains(response, "last_name")
+        self.assertContains(response, "license_number")
```
Comment (Nattalli) [inline tests/test_taxi.py:41]: you can also check the added values with response.data[filed_from_the_answer]

File: tests/test_taxi.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        self.client.force_login(self.admin_user)
+
+    def test_manufacturer_is_register_in_admin(self):
+        url = reverse("admin:taxi_manufacturer_changelist")
+        response = self.client.get(url)
+
+        self.assertEqual(response.status_code, 200)
+
+
+class AminSiteCarTests(TestCase):
```
Comment (Nattalli) [inline tests/test_taxi.py:61]: you have a typo: Here should be admin not amin

File: tests/test_taxi.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        url = reverse("admin:taxi_driver_add")
+        response = self.client.get(url)
+
+        self.assertContains(response, "first_name")
+        self.assertContains(response, "last_name")
+        self.assertContains(response, "license_number")
+        self.assertContains(response, "Additional info")
+
+
+class AminSiteManufacturerTests(TestCase):
```
Comment (Nattalli) [inline tests/test_taxi.py:45]: and here amin

File: tests/test_taxi.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+import os.path
+
+from taxi.models import Car, Manufacturer
+
+
+class AminSiteDriverTests(TestCase):
```
Comment (Nattalli) [inline tests/test_taxi.py:9]: and here

File: tests/test_taxi.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    def setUp(self) -> None:
+        self.client = Client()
```
Comment (Nattalli) [inline tests/test_taxi.py:11]: for not typing this every time you can call `client = test.APIClient()` at once before the starting of your test classes. But I am not sure that it is a best practice

File: tests/test_taxi.py [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    def test_license_number_in_driver_change(self):
+        url = reverse("admin:taxi_driver_change", args=[self.driver.id])
+        response = self.client.get(url)
+
+        self.assertContains(response, self.driver.license_number)
+
+    def test_additional_info_fields_in_driver_add(self):
+        url = reverse("admin:taxi_driver_add")
+        response = self.client.get(url)
+        response[]
```
Comment (danylott) [inline tests/test_taxi.py:38]: Remove this please

**Review by danylott — APPROVED** [review]

**Pull request merged**